eventually/abstractmodel/models.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+from utils.utils import LOGGER
+
+class Abstract(models.Model):
+    def __str__(self):
+        """
+        Magic method is redefined to show all information about object.
+
+        :return: all info about object
+
+        """
+        return str(self.to_dict())[1:-1]
+
+    def __repr__(self):
+        """
+        This magic method is redefined to show class and id of object.
+
+        :return: class, id
+
+        """
+        return f'{self.__class__.__name__}(id={self.id})'
+
+    def to_dict():
```

Comment:
maybe we have to use `@abc.abstractmethod` decorator?

Reply:
done